massa-execution-worker/src/interface_impl.rs

```diff
@@ -690,11 +690,11 @@ impl Interface for InterfaceImpl {
     /// A list of keys (keys are byte arrays)
     ///
     /// [DeprecatedByNewRuntime] Replaced by `get_op_keys_wasmv1`
-    fn get_op_keys(&self) -> Result<Vec<Vec<u8>>> {
+    fn get_op_keys(&self, prefix_opt: Option<&[u8]>) -> Result<Vec<Vec<u8>>> {
         // TODO return BTreeSet<Vec<u8>>
 
-        // TODO add prefix
-        let prefix: &[u8] = &[];
+        let prefix: &[u8] = prefix_opt.unwrap_or_default();
+        // let prefix: &[u8] = Default::default();
```

Comment:
this kind of comments are only causing confusion to new readers and should always be removed

Reply:
Done